rasa/cli/export.py

```diff
@@ -173,7 +173,10 @@ def export_trackers(args: argparse.Namespace) -> None:
     Args:
         args: Command-line arguments to process.
     """
-    asyncio.run(_export_trackers(args))
+    # we can replace it with asyncio.run(..) but it doesn't work in Python 3.9
+    # see https://github.com/virtool/virtool-workflow/issues/55#issuecomment-733164513
+    loop = asyncio.get_event_loop()
```

Comment:
@usc-m Pulling you in here as you have done the switch to `asyncio.run`. Do you have any further insights on this one?

Reply:
It looks like this is only breaking if you use a particular version of `uvloop` with `asyncio` in Python 3.9 (specifically anything prior to uvloop v0.15.0 released in Feb 2021, which is after the link here)? What was the particular thing that caused this to break?